elasticdl/python/common/args.py

```diff
@@ -280,6 +280,14 @@ def add_common_params(parser):
         default=False,
         help="True for Go-based PS, False for Python-based PS",
     )
+    parser.add_argument(
+        "--debug_params",
```

Comment:
`aux_params` is better.
auxiliary parameters for misc purposes such as debugging.

Reply:
Done